src/transformState.js

```diff
@@ -1,11 +1,31 @@
 'use strict';
 
 /**
- * @param {Object} state
- * @param {Object[]} actions
- */
+   * @param {Object} state'
+   * @param {Object[]} actions
+   */
```

Comment:
why are you changing the dock to a function?

Reply:
hmm, sory, I don`t know, I took it from the last task